src/library/Uno.Material/Styles/Controls/Button.xaml

```diff
@@ -33,13 +34,23 @@
 	<SolidColorBrush x:Key="ButtonTextLabelColor"
 					 Color="{ThemeResource PrimaryColor}" />
 
+	<!-- Floating Action Button Variables -->
+	<SolidColorBrush x:Key="FabBackgroundColor"
+					 Color="{StaticResource SecondaryColor}" />
+	<SolidColorBrush x:Key="FabIconColor"
+					 Color="{StaticResource OnSurfaceColor}" />
+	<SolidColorBrush x:Key="FabLabelColor"
+					 Color="{StaticResource OnSurfaceColor}" />
+	<CornerRadius x:Key="FabBorderRadius">23</CornerRadius>
+	<x:Double x:Key="FabLargeAreaSize">56</x:Double>
```

Comment:
Is there a way to make it a multiplication of the radius ?
@davidjohnoliver  @agneszitte-nventive @Massimo37  looking for ideas =D
We could do a MultiplyConverter and Binding RelativeSourceTemplatedParent using that converter but it seems quite hackish

Reply:
do I wait on this?

Reply:
no need to wait @NicolasChampagne We can improve later.